mlflow/sagemaker/container/__init__.py

```diff
@@ -25,7 +25,7 @@
 
 MODEL_PATH = "/opt/ml/model"
 
-DEPLOYMENT_CONFIG_KEY_FLAVOR_NAME = "deployment_flavor_name"
+DEPLOYMENT_CONFIG_KEY_FLAVOR_NAME = "MLFLOW_DEPLOYMENT_FLAVOR_NAME"
```

Comment:
This environment variable is used to communicate between the sagemaker deploy process and the container itself, so this change means that deploy requires containers to be on MLflow 1.0. I documented this as part of the release note section.